### ISSUE TITLE
Feature request: Unicode icons for unittest

### ISSUE THREAD

**Issue by thekid** [xp-framework/core#55]
Use  ✓ for successful builds and  ✗ for errors.

**Comment by thekid** [comment]
```diff
diff --git a/src/main/php/xp/unittest/DefaultListener.class.php b/src/main/php/xp/unittest/DefaultListener.class.php
index 49d7fc4..64a0663 100755
--- a/src/main/php/xp/unittest/DefaultListener.class.php
+++ b/src/main/php/xp/unittest/DefaultListener.class.php
@@ -161,9 +161,11 @@ class DefaultListener extends \lang\Object implements TestListener, ColorizingLi
     }

     $this->out->writeLinef(
-      "\n%s%s: %d/%d run (%d skipped), %d succeeded, %d failed%s",
-      $this->colored ? ($fail ? "\033[41;1;37m" : "\033[42;1;37m") : '',
-      $fail ? 'FAIL' : 'OK',
+      "\n%s: %d/%d run (%d skipped), %d succeeded, %d failed%s",
+      ($this->colored
+        ? ($fail ? "\033[41;1;37m✗" : "\033[42;1;37m✓")
+        : ($fail ? 'FAIL' : 'OK')
+      ),
       $result->runCount(),
       $result->count(),
       $result->skipCount(),
```

**Comment by thekid** [comment]
## Success
![quick 61](https://cloud.githubusercontent.com/assets/696742/7332665/fb176980-eb4a-11e4-8557-6b809c70bd83.png)

## Failure
![quick 60](https://cloud.githubusercontent.com/assets/696742/7332666/fc7e179c-eb4a-11e4-80ee-e1345825a270.png)

**Comment by kiesel** [comment]
:+1:

**Comment by thekid** [comment]
Implemented in xp-framework/core@569807d
